src/database/schema/ChatSchema.ts

```diff
@@ -198,7 +211,7 @@ class ChatSchema implements BaseDbSchema {
       channelId: json?.channel_id,
       userId: json?.message?.user?.id,
       message: json?.message?.text || json?.message?.message,
-      type: json?.message?.type,
+      type: json?.message?.message_type ?? json?.message?.type,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change seems to be adding a fallback for the `type` field to use `json?.message?.message_type` if `json?.message?.type` is undefined. This looks fine as long as the `message_type` and `type` fields are intended to serve the same purpose.


<!-- This is an auto-generated comment by OSS CodeRabbit -->